package.json

```diff
@@ -12,52 +42,18 @@
     "write",
     "manipulation"
   ],
+  "license": "WTFPL",
```

Comment:
Please revert license change.

Reply:
I thought I left it the same as it was? I just moved the license field with the other metadata fields.

Yeah here's the license from before I forked it: https://github.com/jDataView/jDataView/blob/0592e6d6cd31b26cf1b3dcacd69b16f29c5f5bca/package.json#L75

Reply:
Oh I just found #86 - the branch I forked from must have been a bit outdated. I've changed it.